Docs/ContributorsGuide.md

```diff
@@ -3,7 +3,7 @@
 Here are a couple of things to make your experience in the community more enjoyable. 
 
 ## Coding
-* Please use the same code style as the rest of the codebase. Visual Studio should pick up most of it from `.editorconfig`. 
+* Please use the same code style as the rest of the codebase. Visual Studio should pick up most of it from `.editorconfig`. You can autoformat the file with the sequence Ctrl+K, Ctrl+D.
```

Comment:
Is this intended or some tray changes?

Reply:
It is intended. It is the second point in the commit message:
> An improvement to the Contributors Guide. It was too small for its
own commit.

In general, I put things in their own commits, but the change was so small that I decided to put it in the same one. Since there is confusion about that, I'll prefer separating the changes from now on.

Reply:
ah sorry, i was going over it quickly on my phone and didn't read the full description. And this stood out. Was just double checking in a lazy way :)